AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginGen2GraphQLTests/README.md

```diff
@@ -67,7 +67,154 @@ const schema = a.schema({
     })
     .identifier(["id", "content"])
     .authorization((allow) => [allow.publicApiKey()]),
-});
+    
+    // Gen2_1
+    // https://docs.amplify.aws/swift/build-a-backend/data/data-modeling/add-fields/#specify-a-custom-field-type
+    Location1: a.customType({
+      lat: a.float(),
+      long: a.float(),
+    }),
+    Post1: a.model({
+      location: a.ref('Location1'),
+      content: a.string(),
+    }),
+    User1: a.model({
+      lastKnownLocation: a.ref('Location1'),
+    }),
+
+    // Gen2_2
+    // https://docs.amplify.aws/swift/build-a-backend/data/data-modeling/add-fields/#specify-an-enum-field-type
+    PrivacySetting2: a.enum([
+      'PRIVATE',
+      'FRIENDS_ONLY',
+      'PUBLIC'
+    ]),
+    Post2: a.model({
+      content: a.string(),
+      privacySetting: a.ref('PrivacySetting2'),
+    }),
+    Video2: a.model({
+      privacySetting: a.ref('PrivacySetting2'),
+    }),
+
+    // Gen2_3
+    // https://docs.amplify.aws/swift/build-a-backend/data/data-modeling/relationships/#model-one-to-many-relationships
+    Member3: a.model({
+        name: a.string().required(),
+        // 1. Create a reference field
+        teamId: a.id(),
+        // 2. Create a belongsTo relationship with the reference field
+        team: a.belongsTo('Team3', 'teamId'),
+    })
+    .authorization(allow => [allow.publicApiKey()]),
+
+    Team3: a.model({
+        mantra: a.string().required(),
+        // 3. Create a hasMany relationship with the reference field
+        //    from the `Member`s model.
+        members: a.hasMany('Member3', 'teamId'),
+    })
+    .authorization(allow => [allow.publicApiKey()]),
+
+    // Gen2_4 - Model a "one-to-one" relationship
+    // https://docs.amplify.aws/swift/build-a-backend/data/data-modeling/relationships/#model-a-one-to-one-relationship
+    Cart4: a.model({
+        items: a.string().required().array(),
+        // 1. Create reference field
+        customerId: a.id(),
+        // 2. Create relationship field with the reference field
+        customer: a.belongsTo('Customer4', 'customerId'),
+    }),
+    Customer4: a.model({
+        name: a.string(),
+        // 3. Create relationship field with the reference field
+        //    from the Cart model
+        activeCart: a.hasOne('Cart4', 'customerId')
+    }),
+    
+    // Gen2_41 -Model multiple relationships between two models
+    // http://localhost:3000/swift/build-a-backend/data/data-modeling/relationships/#model-multiple-relationships-between-two-models
```

Comment:
nit: change the local url to docs url

Reply:
thanks, i'll merge this PR in first and update this line in the other one https://github.com/aws-amplify/amplify-swift/pull/3705 to avoid rebasing since that one and is merging into this branch and also has some minor changes to do